stripe-ui-core/src/main/java/com/stripe/android/uicore/elements/AddressElementUiRegistry.kt

```diff
@@ -0,0 +1,22 @@
+package com.stripe.android.uicore.elements
+
+import com.stripe.android.uicore.address.AddressSchemaRegistry
+import com.stripe.android.uicore.address.transformToElementList
+
+internal class AddressElementUiRegistry(schemaRegistry: AddressSchemaRegistry) {
+    private val defaultElements = schemaRegistry.defaultSchema.schemaElements().transformToElementList(
+        countryCode = schemaRegistry.defaultSchema.countryCode
+    )
+
+    private val elements = schemaRegistry.all.mapValues { (countryCode, schema) ->
+        schema.schemaElements().transformToElementList(countryCode)
+    }
+
+    fun get(countryCode: String?): List<SectionFieldElement>? {
+        return if (countryCode != null) {
+            elements[countryCode]
+        } else {
+            defaultElements
+        }
+    }
+}
```

Comment:
This registry is temporary while we still use `Flow` in `AddressElement`. This will be replaced by inline transformation when using `StateFlow`.